.github/checkgroup.yml

```diff
@@ -25,74 +25,74 @@ subprojects:
       - ".actions/**"
     checks:
       # Note: updates here should be applied to the lightning_lite group
-      - "pl-cpu (macOS-11, pytorch, 3.8, 1.10)"
-      - "pl-cpu (macOS-11, pytorch, 3.9, 1.11)"
-      - "pl-cpu (macOS-11, pytorch, 3.10, 1.12)"
-      - "pl-cpu (macOS-11, pytorch, 3.7, 1.9, oldest)"
-      - "pl-cpu (macOS-11, pytorch, 3.10, 1.13, pre)"
-      - "pl-cpu (ubuntu-20.04, pytorch, 3.8, 1.10)"
-      - "pl-cpu (ubuntu-20.04, pytorch, 3.9, 1.11)"
-      - "pl-cpu (ubuntu-20.04, pytorch, 3.10, 1.11)"
-      - "pl-cpu (ubuntu-20.04, pytorch, 3.10, 1.12)"
-      - "pl-cpu (ubuntu-20.04, pytorch, 3.7, 1.9, oldest)"
-      - "pl-cpu (ubuntu-20.04, pytorch, 3.9, 1.13, pre)"
-      - "pl-cpu (windows-2022, pytorch, 3.9, 1.11)"
-      - "pl-cpu (windows-2022, pytorch, 3.10, 1.11)"
-      - "pl-cpu (windows-2022, pytorch, 3.10, 1.12)"
-      - "pl-cpu (windows-2022, pytorch, 3.7, 1.9, oldest)"
-      - "pl-cpu (windows-2022, pytorch, 3.8, 1.13, pre)"
-      - "pl-cpu (macOS-11, lightning, 3.10, 1.12)"
+      - "pl-cpu (0, macOS-11, pytorch, 3.8, 1.10)"
```

Comment:
can we please have the slow flag be a string ("slow"/"fast") so that it's obvious what it is for. First time someone sees a 0, they won't have an idea what is it for

Reply:
Good point, let me check it... on the other hand slow/fas is confusing... so would be have some other names?

Reply:
That'll work, too!